style.css

```diff
@@ -705,4 +700,54 @@ hr {
     list-style-type: none;
     right: 10px;
   }
+
+  .desktop-nav > ul > li {
+    text-decoration: none;
+    list-style-type: none;
+    font-weight: 500;
+    font-size: 15px;
+    align-items: center;
+    font-style: normal;
+    padding: 1em;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  @keyframes animation {
+    from {
+      border-bottom: none;
+    }
+
+    to {
+      border-bottom: 1px solid #2230d2;
+    }
+  }
+
+  .desktop-nav > ul > li:active {
+    color: #2230d2;
+    cursor: pointer;
+    animation-name: animation;
+    animation-duration: 0.3s;
+  }
+
+  .desktop-nav > ul > li:hover {
+    color: #2230d2;
+    cursor: pointer;
+    animation-name: animation;
+    animation-duration: 0.3s;
+    animation-timing-function: ease-in;
+    animation-fill-mode: forwards;
+  }
+
+  @keyframes slidein {
+    from {
+      margin-left: 100%;
+      width: 30%;
+    }
+
+    to {
+      margin-left: 0%;
+      width: 30%;
+    }
+  }
 }
```

Comment:
- [ ] Good job implementing the desktop view 💯, but please note you still have to maintain a perfect mobile design. Your mobile design now has a horizontal scroll and some of your elements are overflowing. Kindly fix the mobile design to make sure that it remain perfect.